### PR TITLE
feat(BUX-239): adds issue templates

### PR DESCRIPTION
**Pull request by wregulski**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue-template.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+---
+
+**TL;DR**
+Check if your issue answers those question:
+
+- Does my issue clearly describe what exactly is not working properly?
+- Does my issue clearly describe what are my expectations and what is the current output?
+- Did I attach logs/screenshots/errors which ocured?
+- Did I provide information how I run the code/library/application and on which platform/browser?
```
Comment (kuba-4chain) [inline .github/ISSUE_TEMPLATE/issue-template.md]: This could actually be a checklist, but I'm just nitpicking 😄

**Review by kuba-4chain — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/issue-template.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+If you have any logs connected with your problem - please attach them.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Smartphone (please complete the following information):**
```
Comment (dorzepowski) [inline .github/ISSUE_TEMPLATE/issue-template.md]: Maybe just `Environment`, or `System information`?

File: .github/ISSUE_TEMPLATE/issue-template.md (hunk 1 above)
Comment (wregulski) [reply .github/ISSUE_TEMPLATE/issue-template.md]: Done

File: .github/ISSUE_TEMPLATE/issue-template.md (hunk 2 above)
Comment (wregulski) [reply .github/ISSUE_TEMPLATE/issue-template.md]: Done

**Pull request merged**